code/modules/hydroponics/grown/root.dm

```diff
@@ -34,7 +34,7 @@
 /obj/item/reagent_containers/food/snacks/grown/carrot/attackby(obj/item/I, mob/user, params)
 	if(is_sharp(I))
 		to_chat(user, "<span class='notice'>You sharpen the carrot into a shiv with [I].</span>")
-		var/obj/item/kitchen/knife/carrotshiv/Shiv = new /obj/item/kitchen/knife/carrotshiv
+		var/obj/item/kitchen/knife/shiv/carrot/Shiv = new
```

Comment:
Can you add parenthesis while you're here? Its just a standard even if we have no args
```suggestion
		var/obj/item/kitchen/knife/shiv/carrot/Shiv = new ()
```